environment.yml

```diff
@@ -1,24 +1,42 @@
 ---
-# yamllint disable rule:line-length
 name: armory-core
```

Comment:
Changing the `environment.yml` will impact the base image. Not something we want to do with this mod.

Reply:
The changes do effect the base image, but all images continue to build as normal.